hazelcast/src/main/java/com/hazelcast/client/impl/ClientEngineImpl.java

```diff
@@ -534,8 +535,15 @@ public void run() {
     }
 
     @Override
-    public Operation getPostJoinOperation() {
-        return ownershipMappings.isEmpty() ? null : new PostJoinClientOperation(ownershipMappings);
+    public Operation getPreJoinOperation() {
```

Comment:
Is there anything needed for 3.9 compat?

Reply:
I am guessing no. I had to move filtering of dead clients from the node where operation run to the node where operation originated from. 
When 3.9 and 3.10 working together, 
We will have `postjoin` from 3.9 to 3.10 node
and `prejoin` from 3.10 to 3.9 node. 
All should be OK, right?